notebooks/wine_predictor_analysis_report.ipynb

```diff
@@ -594,11 +612,41 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 19,
    "id": "06f40256-f308-4e7f-a3a7-8351360e8165",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Best hyperparameters: {'svc__C': np.float64(1.934556578726365), 'svc__decision_function_shape': 'ovr', 'svc__gamma': 1, 'svc__kernel': 'rbf'}\n",
+      "Best cross-validation score: 0.6481740196078432\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Hyperparameter tuning for the best performing model\n",
+    "param_dist = {\n",
+    "    'svc__C': loguniform(1e-3, 1e3),\n",
+    "    'svc__kernel': ['linear', 'rbf', 'poly'],\n",
+    "    'svc__gamma': ['scale', 'auto', 0.01, 0.1, 1],\n",
+    "    'svc__decision_function_shape': ['ovr', 'ovo'] \n",
+    "}\n",
+    "\n",
+    "svc_tuning = make_pipeline(preprocessor, SVC())\n",
+    "\n",
+    "random_search = RandomizedSearchCV(svc_tuning, param_dist, \n",
+    "                                   n_iter=50, cv=5, n_jobs=-1, random_state=123) #Do we want random_state?\n",
```

Comment:
As per group discussion, I don't think we are going forward with the random_state.
```suggestion
    "                                   n_iter=50, cv=5, n_jobs=-1) #Do we want random_state?\n",
```